### PR TITLE
Created doc: FETCH

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

**Review by jievince — DISMISSED** [review]

File: docs-2.0/3.ngql-guide/7.general-query-statements/4.fetch.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+## OpenCypher Compatibility
+
+This topic applies to nGQL extensions only.
+
+## Fetch vertex properties
+
+### Syntax
+
+```ngql
+FETCH PROP ON {<tag_name> | <tag_name>[, tag_name ...] | *}
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/7.general-query-statements/4.fetch.md:14]: `FETCH PROP ON <tag_name>[, tag_name ...]`

File: docs-2.0/3.ngql-guide/7.general-query-statements/4.fetch.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
++-------------+--------------+
+Got 1 rows (time spent 2933/5931 us)
+```
+
+### Fetch properties of multiple vertices
+
+Specify multiple VIDs (vertex IDs) to fetch properties of multiple vertices. Separate the VIDs with commas.
+
+```ngql
+FETCH PROP ON player "player101", "player102", "player103";
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/7.general-query-statements/4.fetch.md:62]: `nebula> `

File: docs-2.0/3.ngql-guide/7.general-query-statements/4.fetch.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+nebula> FETCH PROP ON serve "player100" -> "team204", "player133" -> "team202";
++-----------------------------------------------------------------------+
+| edges_                                                                |
++-----------------------------------------------------------------------+
+| [:serve "player100"->"team204" @0 {end_year: 2016, start_year: 1997}] |
++-----------------------------------------------------------------------+
+| [:serve "player133"->"team202" @0 {end_year: 2011, start_year: 2002}] |
++-----------------------------------------------------------------------+
+Got 2 rows (time spent 1466/2441 us)
+```
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/7.general-query-statements/4.fetch.md:191]: add a rank example.

Comment (whitewum) [inline docs-2.0/3.ngql-guide/7.general-query-statements/4.fetch.md:191]: add a rank example.

**Review by whitewum — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]
good

**Pull request merged**